cmd/podman/pods/create.go

```diff
@@ -110,7 +109,7 @@ func create(cmd *cobra.Command, args []string) error {
 		return errors.Wrapf(err, "unable to process labels")
 	}
 
-	imageName = config.DefaultInfraImage
+	imageName = ""
```

Comment:
I think that https://github.com/vrothberg/libpod/commit/aa258813c54adf3e63d9d28e7187aa9eddb7a120 is a bit cleaner.  The defaults should come from the client.

Reply:
agree.  Closed this PR in favor of #12250